package/MDAnalysis/coordinates/DCD.py

```diff
@@ -191,11 +191,10 @@ def _read_next_timestep(self, ts=None):
         if ts is None:
             # use a copy to avoid that ts always points to the same reference
             # removing this breaks lammps reader
-            ts = self.ts.copy() # why is this copy required ??
+            ts = self.ts # why is this copy required ??
         frame = self._file.read()
         self._frame += 1
-        ts = self._frame_to_ts(frame, ts)
-        self.ts = ts
```

Comment:
Do the other readers not set self.ts?

Reply:
They other readers don't set self.ts as ts as ts is modified inplace in `_frame_to_ts`